spec/models/meursing_lookup/steps/milk_fat_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+RSpec.describe MeursingLookup::Steps::MilkFat do
+  it_behaves_like 'an answer step', 'milk_fat' do
+    describe '#current_meursing_code_level' do
+      let(:input_answers) do
+        {
+          'starch' => '0 - 4.99',
+          'sucrose' => '0 - 4.99',
+        }
+      end
+
+      let(:tree) do
```

Comment:
maybe
`let(:meursing_codes) do`  ?
...
and 

`eq(meursing_codes['0 - 4.99']['0 - 4.99'])`